packages/core/integration-tests/test/cache.js

```diff
@@ -441,6 +455,21 @@ describe('cache', function() {
                   }),
                 );
 
+                // At the moment, adding a JS config requires restarting the process
+                // due to https://github.com/babel/babel/pull/12211. Simulate this
+                // by clearing the require cache for @parcel/transformer-babel and @babel/core.
+                await workerFarm.callAllWorkers('invalidateRequireCache', [
+                  packageManager.resolveSync(
+                    '@parcel/transformer-babel',
+                    __filename,
+                  )?.resolved,
+                ]);
```

Comment:
Again: https://github.com/parcel-bundler/parcel/pull/5802#discussion_r583896736

Can you merge/rebase this in a different way to make you sure catch everything?